src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/ChimericAlignment.java

```diff
@@ -300,28 +312,32 @@ static boolean isNotSimpleTranslocation(final AlignmentInterval regionWithLowerC
      * But to fully resolve this case, we need other types of evidence, hence should not be the task of this function.
      */
     public boolean isNotSimpleTranslocation() {
-        return isNotSimpleTranslocation(regionWithLowerCoordOnContig, regionWithHigherCoordOnContig, strandSwitch,
-                                        involvesRefPositionSwitch(regionWithLowerCoordOnContig, regionWithHigherCoordOnContig));
+        return !isLikelySimpleTranslocation(regionWithLowerCoordOnContig, regionWithHigherCoordOnContig, strandSwitch);
```

Comment:
I'd remove this method now that's just a simple negation.

Reply:
this method exists for easier usage by other classes (mostly for filtering or branching logic). How about changing it to non-static version `isLikelySimpleTranslocation()`?